sailesh_work/terraform/backend.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {
+    bucket = "devopsremotestate1"
+    key    = "terraform.tfstate"
+    region = "us-east-1"
+  }
```

Comment:
pls use varialbe to refer it --eg--->>var.region

Reply:
We can't use variables in the backend block due to limitations for the backend block